src/duration/mod.rs

```diff
@@ -242,10 +243,53 @@ impl Div<u32> for Duration {
 }
 
 impl Duration {
+    /// Creates a new [`Duration`] from the specified number of whole milliseconds.
+    pub const fn from_millis(millis: u64) -> Self {
+        Self(std::time::Duration::from_millis(millis))
+    }
+
     /// Creates a new [`Duration`] from the specified number of whole seconds.
     pub const fn from_secs(secs: u64) -> Self {
         Self(std::time::Duration::from_secs(secs))
     }
+
+    /// Creates a new [`Duration`] from the specified number of whole minutes.
+    /// Panics if the Duration is higher than [`std::time::Duration`] supports
+    /// (which is 584.9 years). It is recommended to only use this function in `const`
```

Comment:
We might want to update this to the correct value mentioned [here](https://doc.rust-lang.org/std/time/struct.Duration.html#associatedconstant.MAX).

Reply:
Switched in 25a5c8a0b0c0876feb2a6e8e1aecd034a6194bf3